### PR TITLE
fix(2453): adjust UI details for account cell on wallet screen

### PR DESCRIPTION
**Pull request by vinnyhoward**
## **Description**

Adjusting the wallet accounts UI to be the same as the [Figma Design](https://www.figma.com/file/njDnVDROVuIwLbjUjkUYuO/MC-Mobile-v1-Amon-Hen?type=design&node-id=1695-33336&mode=design&t=5DcwE7YmybXP7b4J-0)

UPDATE:
Adjusted `PickerAccount` and `AddressCopy` styling to be the same as this [Figma Design](https://www.figma.com/design/MlBUrNnIXlftX4TSjqiUYg/Core-Components?m=auto&node-id=4222-100137) (new source of truth)

## **Related issues**

Fixes: [#2453](https://github.com/MetaMask/MetaMask-planning/issues/2453)

## **Manual testing steps**

1. Goto wallet home page
2. View changes in Account Picker and Address Copy
3. Make sure the font style changes and colors are the same as [Figma](https://www.figma.com/design/MlBUrNnIXlftX4TSjqiUYg/Core-Components?m=auto&node-id=4222-100137)

## **Screenshots/Recordings**

| Before | After |
|:---:|:---:|
| before|after|

### **Before**

NA

### **After**

NA

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [x] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [x] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by jonybur — DISMISSED** [review]

**Review by EtherWizard33 — DISMISSED** [review]
LGTM

**Comment by brianacnguyen** [comment]
Consulting with design regarding the text variant change

File: app/component-library/components/Pickers/PickerAccount/PickerAccount.tsx [hunk 1]
```diff
@@ -51,7 +51,7 @@ const PickerAccount: React.ForwardRefRenderFunction<
       />
       <View style={styles.accountNameLabel}>
         <Text
-          variant={TextVariant.HeadingSMRegular}
+          variant={TextVariant.BodyMDMedium}
```
Comment (georgewrmarshall) [inline app/component-library/components/Pickers/PickerAccount/PickerAccount.tsx:54]: Can we confirm with design on this change? I see BodySMMedium in the [Figma](https://www.figma.com/design/MlBUrNnIXlftX4TSjqiUYg/Core-Components?node-id=4685-31&node-type=canvas&m=dev)

Screenshot 2024-09-23 at 11 12 34 AM

**Review by georgewrmarshall — COMMENTED** [review]
Left one comment for design regarding correct type style but otherwise LGTM! Really nice PR description and easy to review PR 💯

File: app/component-library/components/Pickers/PickerAccount/PickerAccount.tsx (hunk 1 above)
Comment (brianacnguyen) [reply app/component-library/components/Pickers/PickerAccount/PickerAccount.tsx:54]: @georgewrmarshall I see Body-MD-Medium

**Review by brianacnguyen — APPROVED** [review]

File: app/component-library/components/Pickers/PickerAccount/PickerAccount.tsx (hunk 1 above)
Comment (georgewrmarshall) [reply app/component-library/components/Pickers/PickerAccount/PickerAccount.tsx:54]: I still see `Body-SM-Medium` in the Figma? Definitely agree with large font sizes overall. It improves accessibility does the Figma need to be updated? cc @Akatori-Design @yanrong-chen 
Screenshot 2024-09-25 at 1 13 44 PM

Comment (georgewrmarshall) [reply app/component-library/components/Pickers/PickerAccount/PickerAccount.tsx:54]: Whoops! I was looking at the wrong thing apologies!

**Pull request merged**